libvmi/json_profiles/json_profiles.c

```diff
@@ -100,6 +105,27 @@ status_t vmi_get_symbol_addr_from_json(vmi_instance_t vmi, json_object* json, co
     return vmi->json.handler(json, symbol, NULL, addr, NULL);
 }
 
+status_t vmi_address_to_symbol_from_json(vmi_instance_t vmi, addr_t addr, const char** symbol)
```

Comment:
I'm not a 100% sure we would want a different API entry point for this, why not just use `vmi_translate_v2ksym`?

Reply:
I removed the method and added the ht lookup to the implementation used by `vmi_translate_v2ksym`